examples/reader.raku

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env raku
+
+use lib '.';
```

Comment:
Why this?  If people run this from the example directory, then it wouldn't work.  Or use the installed version.  Which may not have the new features.